docs/source/en/model_doc/dinov2.md

```diff
@@ -25,6 +25,37 @@ The abstract from the paper is the following:
 This model was contributed by [nielsr](https://huggingface.co/nielsr).
 The original code can be found [here](https://github.com/facebookresearch/dinov2).
 
+## Usage tips
```

Comment:
@ArthurZucker WDYT of this addition to the model page?

Reply:
Love it! 
We often have people asking how xx model can be compiled / traced so great to be careful when we add support for this! 😉